src/components/MyKiva/MyKivaCard.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+	<div
+		class="tw-w-full tw-relative tw-rounded tw-bg-cover tw-bg-center tw-select-none journey-card"
+		:class="{ '!tw-bg-left-top': isBgTopAligned }"
+		:style="{ backgroundImage: `url(${bgImage})` }"
+	>
+		<div
+			v-if="tagText"
+			class="tw-absolute tw-bg-secondary tw-rounded tw-px-1.5 tw-py-0.5 tw-text-small tw-left-1.5 tw-top-1.5
+				tw-drop-shadow-sm tw-font-medium"
+		>
+			{{ tagText }}
+		</div>
+		<div class="tw-p-1">
+			<img
+				v-if="imageUrl"
+				:src="imageUrl"
+				class="tw-rounded tw-w-full"
+				loading="lazy"
+			>
+		</div>
+		<div
+			class="
+				tw-w-full
+				tw-bottom-0
+				tw-pb-1.5
+				tw-px-1.5
+				md:tw-pb-2
+				md:tw-px-2
+				tw-align-bottom
+				tw-rounded-b
+			"
+			:class="{
+				'slide-gradient': hasGradient,
+				'tw-absolute': !imageUrl,
+			}"
+			:style="[
+				{ 'height': overlayHeight },
+			]"
+		>
+			<div class="tw-flex tw-flex-col tw-justify-end tw-h-full !tw-gap-1.5">
```

Comment:
It looks like this content needs to be pushed to the bottom. The title and content move down as the image loads. The pop-in only affects the new variation.

Reply:
Yes, if it happens in the app as well, it would be worth a fix. Just wanted to call it out. Otherwise everything looks good!

Reply:
Thanks, I'll check it out.
There is more upcoming work, this PR is mainly to create a new component and replace in current carousel we have (journeys).

Current new variation in Storybook is not precisely final version, we need to replace that image in the app, and some extra tweaks are coming...

Just to be sure, can you record a video just to check what probably I don't see?

And good to ask you, should we consider carousel version as a variation of this?

Reply:
You mean add this new card to a carousel? I don't think we need that work yet.

When it comes to repro, just delete the image from the page, and you'll see the content push up. It's the same general affect of the image not being loaded, since there isn't a default image on the img tag.

Reply:
I was testing by deleting the element, not the image URL, but I saw the same thing as the image was loading.

Oh good question on the image carousel, I don't remember seeing that previously. Could you ask Nathan and Lauren in the team channel if we're only showing one borrower image for now?

Reply:
Ok. I'll try to replicate what you mean about it in upcoming work, thanks!